tests/helpers/test_template.py

```diff
@@ -1110,21 +1104,21 @@ def test_bitwise_and(hass):
             """,
         hass,
     )
-    assert tpl.async_render() == str(8 & 8)
+    assert tpl.async_render() == 8 & 8
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6294088)